Sources/CoreXLSX/Comments.swift

```diff
@@ -36,12 +36,25 @@ public struct Comment: Codable, Equatable {
     case reference = "ref"
     case text
   }
+    
+  public var plain: String? {
+      return self.text.r?.t
```

Comment:
Given the comment on `plain`:

```suggestion
      return self.text.r?.t ?? self.text.plain
```